File: integration-tests/scala/src/test/resources/projects/external-reloadable-artifacts/app/src/main/scala/org/acme/GreetingResource.scala
package org.acme

import jakarta.ws.rs.{GET, Path, Produces}
import jakarta.ws.rs.core.MediaType

import org.acme.lib.Greeting

@Path("/hello")
class GreetingResource {

  @GET
  @Produces(Array[String](MediaType.TEXT_PLAIN))
  def hello(): String = Greeting.hello()

}

File: integration-tests/maven/src/test/resources-filtered/projects/external-reloadable-artifacts/app/src/main/java/org/acme/GreetingResource.java
package org.acme;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

import org.acme.lib.*;

@Path("/hello")
public class GreetingResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String get() {
        return Greeting.hello();
    }
}